### ISSUE TITLE
Linking issue after CocoaPods 1.0.0 update

### ISSUE THREAD

**Issue by AllDmeat** [ryanmaxwell/RMUniversalAlert#19]
So here's what I've done:

1. Updated CocoaPods to stable 1.0.0 release;
2. Removed workspace and Pods folder;
3. Executed `pod install`;
4. Cleaned product and build folder.

After that project won't compile — RMUniversalAlert.m throws error:

> 'UIAlertView+Blocks/UIAlertView+Blocks.h' file not found

If i replace imports of UIAlertView, UIActionSheet and UIAlertController like this — everything compiles just fine:
```
@import UIAlertView_Blocks.UIAlertView_Blocks;
@import UIActionSheet_Blocks.UIActionSheet_Blocks;
@import UIAlertController_Blocks.UIAlertController_Blocks;
```

But that's dirty.

Is there a clean way to fix this? Is it CocoaPods related or what?

Asking you, because such error appeared only on your library.

Thanks. Been using your libs for a long time.

**Comment by juliengomes** [comment]
+1

**Comment by aman-clickapps** [comment]
+1, please fix this please....

**Comment by fkmhrk** [comment]
I updated import sections in Pods/RMUniversalAlert/RMUniversalAlert.m and build process succeeded. 

```
#import "UIAlertView+Blocks.h"
#import "UIActionSheet+Blocks.h"
#import "UIAlertController+Blocks.h"
```

**Comment by esanchezaltran** [comment]
+1

**Comment by kevin-hirsch** [comment]
+1

**Comment by albetjanuar** [comment]
+1

**Comment by ryanmaxwell** [comment]
fixed in 0.8
